README.md

```diff
@@ -69,11 +69,11 @@ Find out more on the <a href="https://layer5.io/community#meshmate">Layer5 commu
 
 ## Contributing to the Meshery.io Website
 
-Before contributing, please review the [Contribution Flow](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md). In the following steps you will set up your development environment, fork and clone the repository, run the site locally, and finally commit, sign-off, and push any changes made for review.
+Before contributing, please review the [Contribution Flow](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md). In the following steps, you will set up your development environment, fork and clone the repository, run the site locally, and finally commit, sign off, and push any changes made for review.
 
 ### 1. Set up your development environment
 
-- _The Meshery site is built using Jekyll - a simple static site generator! You can learn more about Jekyll and setting up your development environment in the [Jekyll Docs](https://jekyllrb.com/docs/)._
+- _The Meshery site is built using Jekyll - a simple static site generator! You can learn more about Jekyll and set up your development environment in the [Jekyll Docs](https://jekyllrb.com/docs/)._
```

Comment:
Let's leave this as-is. You can learn about "setting up your development environment" in the Jekyll Docs, however, you can't "set up your development environment in the Jekyll Docs". You setup your development environment on your local machine.